solutions/remotemonitoring/armtemplates/local.json

```diff
@@ -243,7 +243,10 @@
         "consumerGroupName": "streamingJobsConsumerGroup",
         "streamingJobsOutputDBName": "pcs-iothub-stream",
         "storageApiVersion": "2017-06-01",
-        "storageResourceId": "[resourceId('Microsoft.Storage/storageAccounts/',  parameters('storageName'))]"
+        "storageResourceId": "[resourceId('Microsoft.Storage/storageAccounts/',  parameters('storageName'))]",
+        "azureMapsName": "[concat(take(uniqueString(subscription().subscriptionId, resourceGroup().id, parameters('solutionName')), 5), '-map')]",
+        "azureMapsVersion": "2017-01-01-preview",
+        "azureMapsPricingSpec": "S0"
```

Comment:
nit: azureMapsSku maybe more straightforward.

Reply:
This naming pattern comes from our other templates which comes from Azure Maps sample from portal deployment. Lets take this PR and if we want to change all the templates we should do it separately.